lib/ansible/modules/network/cnos/cnos_vlan.py

```diff
@@ -23,532 +21,398 @@
 # Module to send VLAN commands to Lenovo Switches
 # Overloading aspect of vlan creation in a range is pending
 # Lenovo Networking
-#
+
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
 
 ANSIBLE_METADATA = {'metadata_version': '1.1',
                     'status': ['preview'],
-                    'supported_by': 'community'}
```

Comment:
Unless something has changed, this is still `'supported_by': 'community'`

Reply:
This is just a copy paste issue, I missed to add it. Will add it now